spec/std/env_spec.cr

```diff
@@ -2,6 +2,23 @@ require "spec"
 require "./spec_helper"
 
 describe "ENV" do
+  # preserves the existing environment for each spec
+  # to avoid potential circular definitions, this has to use the system methods
+  # directly, rather than `ENV` or `with_env`
```

Comment:
**typo:** Adding some punctuation for enhanced legibility.
```suggestion
  # Preserves the existing environment for each spec.
  # To avoid potential circular definitions, this has to use the system methods
  # directly, rather than `ENV` or `with_env`.
```